.github/workflows/submitBeta.yml

```diff
@@ -0,0 +1,127 @@
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version_name
+          value: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+      - name: Update manifest.json version
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version
+          value: ${{ github.event.inputs.release }}
+      - name: Build extension
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - run: yarn install && yarn build:freighter-api && yarn build:extension:production --define 'process.env.AMPLITUDE_KEY="${{ secrets.AMPLITUDE_KEY }}"' --define 'process.env.SENTRY_KEY="${{ secrets.SENTRY_KEY }}"'
+      - name: Install zip
+        uses: montudor/action-zip@c25e01d7489d0274569440a2f0281b4569df16bc #v0.1.1
+      - name: Zip extension build
+        run: zip -qq -r ./build.zip *
+        working-directory: ./extension/build
+      - name: Commit files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add -A
+          git commit -m "docs(): bumping release to ${{ github.event.inputs.version }}"
+          git tag ${{ github.event.inputs.version }}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #v3.10.1
+        with:
+          title: Bump versions to ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+      - name: Create GitHub Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          tag_name: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          release_name: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          body: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+          draft: false
+          prerelease: true
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./extension/build/build.zip
+          asset_name: build-${{ github.event.inputs.version }}.zip
+          asset_content_type: application/zip
+      - name: Build extension for Firefox
+        id: firefox-build
+        uses: kewisch/action-web-ext@a28b4ba2f491cc36c67a714a716ca54218440056 #v1
+        with:
+          cmd: build
+          source: ./extension/build
+          filename: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}.xpi
+      - name: Sign extension for Firefox
```

Comment:
❓ related to my Q above, is this submitting to firefox? In submitProduction.yml the job name is `Submit extension to Firefox` even though here it says `Sign`, but the jobs look similar?

Reply:
So these are very similar, but the big difference here is `channel: unlisted`. In submitProduction.yml, we're using `channel: listed`.

As I alluded to above, what this means is that submitProduction.yml is just sending our zipped build folder to Firefox to sign, generate, and then host the extension. 

I'm calling this `Sign extension for Firefox` as we're *just* having Firefox sign our .xpi file and then return it to us (used in the Upload Release Asset step). 

These look really similar (and maybe the API shouldn't call both cmds `sign`, but that's on this API's designer 😄  ) but they actually are doing different things for us